modules/microshift-certificate-lifetime.adoc

```diff
@@ -17,24 +17,41 @@ An example of a long-lived certificate is the client certificate for `system:adm
 
 [id="microshift-certificate-rotation_{context}"]
 == Certificate rotation
-As certificates age, {product-title} can be restarted to rotate certificates. A certificate that is close to expiring might also automatically cause a restart. Read the following situation overviews to understand the actions at each moment in time:
+Certificates that are expired or close to expiry need to be rotated to ensure continued {product-title} operation. When {product-title} restarts, certificates that are close to expiring are rotated. A certificate that has expired can also cause an automatic {product-title} restart to perform the rotation.
+
+[NOTE]
+====
+If the rotated certificate is a Certificate Authority, all of the certificates it signed rotate.
+====
+
+[id="microshift-st-certificate-rotation_{context}"]
+=== Short-term certificates
+The following situations describe {product-title} actions during short-term certificate lifetimes:
 
 . Green zone:
 .. When a short-term certificate is 5 months old, no rotation occurs.
-.. When a long-term certificate is 8.5 years old, no rotation occurs.
 
 . Yellow zone:
 .. When a short-term certificate is 8 months old, it is rotated when {product-title} starts or restarts.
-.. When a long-term certificate is 9 years old, it is rotated when {product-title} starts or restarts.
 
 . Red zone
 .. When a short-term certificate is 8 months old, {product-title} restarts to rotate and apply a new certificate.
-.. When a long-term certificate is 9 years old, {product-title} restarts to rotate and apply a new certificate.
 
-[NOTE]
-====
-If the rotated certificate is a Certificate Authority, all of the certificates it signed rotate.
-====
+.Graph of {product-title} short-term certificate validity.
+image::324_RHbM_Certificate_Rotation_0323_short-term.png[<{product-title} bar graph of certificate timelines>]
+
+[id="microshift-lt-certificate-rotation_{context}"]
+=== Long-term certificates
+The following situations describe {product-title} actions during long-term certificate lifetimes:
+
+. Green zone:
+.. When a long-term certificate is 8.5 years old, no rotation occurs.
+
+. Yellow zone:
+.. When a long-term certificate is 9 years old, it is rotated when {product-title} starts or restarts.
+
+. Red zone
+.. When a long-term certificate is 9 years old, {product-title} restarts to rotate and apply a new certificate.
```

Comment:
@ggiguash @ShaunaDiaz This confuses me, too. 
In the yellow zone, a 9-year-old cert requires a manual restart to rotate.  
In the red zone, a 9-year-old cert auto restarts.  Yet the image shows _Restart to rotation_ at 10 years (and no mentions of the color zones.

Reply:
@mburke5678 I can fix that. The previous image was stoplight; Jess updated to accessible colors and I forgot to change the language on those to drop the colors.